### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by mesielepush**
Pull request to merge the feature branch to Development.

### CODE REVIEW

**Comment by mesielepush** [comment]
All linters passed, internal logic works and UI is beautiful.

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    }
+});
+var navTrigger = document.getElementsByClassName('navTrigger')
+navTrigger.click(function () {
+
+    this.toggleClass('active');
+    document.getElementById("mainListDiv").toggleClass("show_list");
+    document.getElementById("mainListDiv").fadeIn();
+
+}); */
```
Comment (mcrd25) [inline src/index.js]: Please remove commented out code. 
As can be seen from [this screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)
It is better to delete commented code. If you need to review a previous version of your code, use git history. 🙂

**Review by mcrd25 — CHANGES_REQUESTED** [review]
# Changes Required :mag:

Hi @mesielepush :wave: 


Good job! Just one tiny change to make! :arrow_down:  :smile: 

***Please makes changes and submit another review request.
If you have any questions, let me know here or on slack and I will be glad to clarify with you. You can also reach me and connect with me on my social media or email.***

*Happy coding 😄,*
**[--Maya](https://github.com/mcrd25)**
[GitHub](https://github.com/mcrd25) [Twitter](https://twitter.com/mcrd25) [linkedin](https://www.linkedin.com/in/mayadouglas/) [GitLab](https://gitlab.com/mcrd25)

---
*P.S. all requested changes not marked by [OPTIONAL] or are not positive comments are **REQUIRED***

**Comment by mesielepush** [comment]
REady, no more commented code on the index.js

**Review by mcrd25 — APPROVED** [review]
# Changes Approved :heavy_check_mark: 

Hi @mesielepush 

All good now! You may now merge and move on! :+1: :next_track_button:

**Pull request merged**